liver_imaging_analysis/engine/engine.py

```diff
@@ -135,12 +149,54 @@ def get_metrics(self, metrics_name, **kwargs):
                 parameters of metrics, if exist.
         """
         metrics = {
-            "accuracy" : Accuracy,
-            "dice" : DiceMetric,
-            "jaccard" : MeanIoU,
+            "accuracy": Accuracy,
+            "dice": DiceMetric,
+            "jaccard": MeanIoU,
         }
         return metrics[metrics_name](**kwargs)
 
+    def get_hparams(self, network_name):
+        """
+        used to return a hyperparameters dictionary specific to each network
+        ----------
+            network_name: str
+                name of network to fetch from dictionary
+                should be chosen from: '3DUNet','3DResNet','2DUNet'
+        """
+        hparams = {
+            "monai_2DUNet": {
+                "spatial_dims": config.network_parameters["spatial_dims"],
+                "num_res_units": config.network_parameters["num_res_units"],
+                "bias": config.network_parameters["bias"],
+                "norm": config.network_parameters["norm"],
+                "dropout": config.network_parameters["dropout"],
+                "batch_size": config.training["batch_size"],
+                "optimizer": config.training["optimizer"],
+                "lr_scheduler": config.training["lr_scheduler"],
+                "loss_name": config.training["loss_name"],
+                "metrics": config.training["metrics"],
+                "shuffle": config.training["shuffle"],
+            }
+        }
+        return hparams[network_name]
+
+    def Hash(self, text: str):
```

Comment:
This doesn't look like an engine function, more of a utility one. I've commented on this previously.

Reply:
Again, these modifications are from an unmerged branch, right?

Reply:
you are right